superset/assets/src/components/ModalTrigger.jsx

```diff
@@ -82,31 +82,31 @@ export default class ModalTrigger extends React.Component {
       'btn btn-default btn-sm': this.props.isButton,
     });
     if (this.props.isButton) {
-      return (
+      return [
```

Comment:
do you think we should use `React.Fragment`s here? I'm not sure of the advantages of them vs arrays (maybe `<>...</>` syntax is supported with our build config?)

Did you check the usage as a `MenuItem`? I was wondering if the parent of a `MenuItem` (prob a `<ul />`) would care if it had a modal / non-`MenuItem` (prob an `<li />`) child . This might not be a problem if the modal renders into a portal (not the parent dom container) since it wouldn't create an invalid DOM structure.

Reply:
framents vs arrays described [here](https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html#what-are-fragments). Not sure it matters too much.

Reply:
@williaster Thanks! was reading the article before. and Indeed In think in this case, `<Fragment />` makes more sense. 

> Children in an array must have a key to prevent React’s key warning.

Reply:
@williaster  and yes, I checked the `MenuItem` :)